pkgs/by-name/ou/outfly/package.nix

```diff
@@ -0,0 +1,87 @@
+{
+  lib,
+  fetchFromGitea,
+  rustPlatform,
+  makeDesktopItem,
+  pkg-config,
+  libxkbcommon,
+  alsa-lib,
+  libGL,
+  vulkan-loader,
+  wayland,
+  libXrandr,
+  libXcursor,
+  libX11,
+  libXi,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "outfly";
+  version = "0.14.0";
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "outfly";
+    repo = "outfly";
+    rev = "v${version}";
+    hash = "sha256-FRvu3FgbT3i5888ll573nhb7naYx04Oi8nrcfgEHxUo=";
+  };
+
+  buildNoDefaultFeatures = true;
+  buildFeatures = [
+    "embed_assets"
+    "x11"
+    "wayland"
```

Comment:
Would having enable flags for X11 and Wayland be useful? (Non-blocking, might be future work)

Reply:
This pull-request originally had feature flags for x11, wayland, openGL, and vulkan, but those were removed to simplify the package.
For this and other reasons the package was built with ''noDefaultFeatures" and required explicitly enabling features.

The current release does not require explicitly selecting features, since both game and pull-request changed since it was first submitted, and just happened to still be there